Actions/AL-Go-Helper.ps1

```diff
@@ -606,7 +609,7 @@ function AnalyzeRepo {
     $settings.appDependencies + $settings.testDependencies | ForEach-Object {
         $dep = $_
         if ($dep -is [string]) {
-            # TODO: handle pre-settings
+            # TODO: handle pre-settings - documentation pending
```

Comment:
missing code

Reply:
Yeah - I think that functionality can wait and maybe we can delete the section